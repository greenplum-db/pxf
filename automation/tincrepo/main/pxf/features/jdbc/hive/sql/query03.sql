-- @description query03 for JDBC Hive query with timestamp filter
SELECT s1, n1, tm FROM pxf_jdbc_hive_types_table WHERE tm > '2013-07-23 21:00:01' ORDER BY n1;
