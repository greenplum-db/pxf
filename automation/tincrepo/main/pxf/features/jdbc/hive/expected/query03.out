-- @description query03 for JDBC Hive query with timestamp filter
SET datestyle TO 'ISO, MDY';
SELECT s1, n1, tm FROM pxf_jdbc_hive_types_table WHERE tm > '2013-07-23 21:00:01' ORDER BY n1, s1;
         s1          | n1 |         tm          
---------------------+----+---------------------
 row11               | 11 | 2013-07-23 21:00:05
 row12_text_null     | 11 | 2013-07-23 21:00:05
 row14_double_null   | 11 | 2013-07-23 21:00:05
 row17_real_null     | 11 | 2013-07-23 21:00:05
 row18_bigint_null   | 11 | 2013-07-23 21:00:05
 row19_bool_null     | 11 | 2013-07-23 21:00:05
 row20_tinyint_null  | 11 | 2013-07-23 21:00:05
 row21_smallint_null | 11 | 2013-07-23 21:00:05
 row22_date_null     | 11 | 2013-07-23 21:00:05
 row23_varchar_null  | 11 | 2013-07-23 21:00:05
 row24_char_null     | 11 | 2013-07-23 21:00:05
 row15_decimal_null  | 12 | 2013-07-24 21:00:05
 row13_int_null      |    | 2013-07-23 21:00:05
(13 rows)

